### DISCUSSION TITLE
MEETING: esites Working Group - 7th December 2022 @ 11:00am

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#721] [MEETINGS]
# Agenda
## Quick review items:
- [Open Issues](https://github.com/fieldmuseum/EMu-Documentation/issues?q=is%3Aissue+esites+is%3Aopen)
- [Q&As](https://github.com/fieldmuseum/EMu-Documentation/discussions/categories/q-a?discussions_q=esites+category%3AQ%26A)

## Discussion Topics
- Sites Record Classification
| Count | Value |
| 80046 |	Aquatic |
| 24256 | Cultural |
| 568107 | Terrestrial |
| 50 | (empty) |
| 672459 | Total |

**Comment by rondlg** [comment]
# Meeting Notes

## Attendees
SG, JJ, KW, KGL, RD, RC, JM, SM, RS, MT, AF, LS, CP, CN, WG, PM, 

## Record Classification
Background: Existing values in this field are legacy terms whose purpose related to tab & field switching + summary string. Their use dates back to when sites had a dual role including habitat related concepts. During the standardization process these habitat related fields were moved to the collections event module.

If the main point of the Classification field is to drive the summary string. As we share data more and more Is this still something that is relevant/necessary? Also what happens when it is not clear if the site is actually on shore or truly aquatic? What about intertidal, this depends on the time of day. What are the implications of continuing this way and if we change what do we change to?


**Inverts**:
List view = current workaround where summary string might miss a piece but it's cumbersome.

**Anthro**:
Usually doesn’t affect / most records = terrestrial, but some shipwreck – possibly “Cultural” currently, but should also reflect Aquatic

**Mammals**:  
- auto-fills w/ Terrestrial / doesn’t otherwise edit

**Error radiuses** 
- don’t usually include aquatic regions.
- Country level v Precise Localities?
- Political v Biological?  [/Cultural?]

**Continent Unknown - International Waters discussion**
- https://github.com/fieldmuseum/EMu-Documentation/issues/6

**DECISION: Replace "International Waters"**

**Question: Does anyone even use the Classification field to search on?**
- There is also a classification in collection event which seems more useful and makes sense.

**Hierarchy / Parent records**
- Facilitates deduplicating names/localities
- How to handle localities where top-level parent results in unexpected hierarchy &/or summary
- - e.g. Tampa Bay -> Gulf of Mexico -> N. Atlantic  
- and/or/vs possibly-more-familiar terrestrial hierarchy
- Question: [Why no Freshwater hierarchy – was TBD / after work groups got thru other hierarchies]

**SummaryStrings**
- How articulate logic for when who wants what in summary [+tabs/etc]
- Rework SummaryStrings based on these new classifications
- - Consider including the rank of the site in Summary string?
- - - Of interest for Insects

**DECISION: Rework the classification to use:**
- Political
- Geospatial
- Cultural

**Homework**:
1) Inverts Team
- make a 1st draft of logic for the new Classification Values https://github.com/fieldmuseum/EMu-Documentation/issues/722
- Propose a replacement term for International Waters https://github.com/fieldmuseum/EMu-Documentation/issues/723

2) EMu Admin
- Search defaults / clear/update old ones
- Tab switching 
- Setup a  Sites workshop / review of creating/editing/cleaning records 
- - [esites records How To...](https://docs.google.com/document/d/1YA87zYvahTko2McaCWmTofg7XBdjKAbfZjJSG6KfS6Q)

**Comment by rondlg** [comment]
Homework follow-up prior to esites WG 9 May 2023
- esites workshop was held: https://github.com/fieldmuseum/EMu-Documentation/discussions/759
- [esites basic training presentation](https://docs.google.com/presentation/d/1SCVwma_vWnFJHeAjspoN8uS9-Zq8PBrz)

**Comment by dangeloje** [comment]
On October 9, members of Invertebrates (Robin and John), Insects (Jessica and Beka), and Fishes (Caleb, Susan, and Andrew), the three divisions that use large numbers of aquatic site records, met to discuss whether the distinction between aquatic and terrestrial site records mattered to us. We concluded that we did not need the distinction to be maintained.
